mantidproject/mantid
# Add Alpha (Balance Parameter) To ALC Interface

~~**Merge #29847  first**~~ **Merged**

**Description of work.**
Added new option to ALC interface to set alpha value

**To test:**
1. Open ALC
2. Load some single period data and check alpha is available (MUSR 62260)
3. Load some multi period data and check alpha is not available (MUSR 15189)

Fixes #29798 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
